enterprise/backend/src/metabase_enterprise/advanced_config/models/pulse_channel.clj

```diff
@@ -0,0 +1,36 @@
+(ns metabase-enterprise.advanced-config.models.pulse-channel
+  (:require [clojure.string :as str]
+            [flatland.ordered.set :as ordered.set]
+            [metabase.models.setting :as setting :refer [defsetting]]
+            [metabase.public-settings.premium-features :as premium-features]
+            [metabase.util :as u]
+            [metabase.util.i18n :refer [deferred-tru tru]]))
+
+(defsetting subscription-allowed-domains
+  (deferred-tru "Allowed email address domain(s) for new DashboardSubscriptions and Alerts. Does not affect existing subscriptions.")
+  :visibility :public
+  :type       :csv
+  ;; maintain the order of the domains so it doesn't shift around during the roundtrip after someone updates the list.
+  :getter     #(not-empty (into (ordered.set/ordered-set) (setting/get-csv :subscription-allowed-domains))))
```

Comment:
this might be easier/clearer as `#(some-> (setting/get-csv :subscription-allowed-domains) ordered.set/into-ordered-set)`

Reply:
Yeah I guess Settings can't be empty strings now that I think about it (or if they are they're interpreted as `nil`)

Reply:
I ended up making this a `:string` Setting since it's inconvenient to work with on the FE if it's `:csv` since it comes back as an array in the API